src/utils/threshold.ts

```diff
@@ -69,7 +69,21 @@ export function extractStatusCounts(profile: ContextualizedProfile, severity?: s
   for (const c of profile.contains.filter(control => control.extendedBy.length === 0)) {
     const control = c.root
     const status: ControlStatus = control.hdf.status
-    const controlSeverity: Severity = control.hdf.severity
+    const impact = control.data.impact
+
+    // using impact here is more reflective of how important these controls are
+    let controlSeverity: Severity
```

Comment:
probably better to have this be an exported function from inspecjs instead of just randomly here in the saf cli

Reply:
done